src/dsp_tools/utils/stack_handling.py

```diff
@@ -91,7 +93,9 @@ def start_stack(
         ("knora-ontologies/standoff-data.ttl", "http://www.knora.org/data/standoff"),
         ("knora-ontologies/salsah-gui.ttl", "http://www.knora.org/ontology/salsah-gui"),
         ("test_data/all_data/admin-data-minimal.ttl", "http://www.knora.org/data/admin"),
-        ("test_data/all_data/permissions-data-minimal.ttl", "http://www.knora.org/data/permissions")
+        ("test_data/all_data/permissions-data-minimal.ttl", "http://www.knora.org/data/permissions"),
```

Comment:
The e2e tests rely on `anything`, so I had to add `anything` here.

Reply:
I'm not sure how the test setup is now, but I wouldn't add this dependency to DSP-TOOLS. It might break if we change this dataset in DSP-API.
Would it be possible to create new tests that don't rely on the "anything" project?

Reply:
Well, until now it worked quite well... If it breaks some day, I'm of course eager to fix it (or rewrite the tests). But rewriting the tests just because it **might** break one day would be a bit demotivating. Because it's quite a lot of testing code that uses the anything data...

Reply:
I agree that this dependency is not ideal, but I also think it's not urgent. Over time it would be good to work towards having less e2e tests (only as many as it really needs) and more unit tests. Then you would also have less tests depending on the `anything` project.